.travis.yml

```diff
@@ -3,30 +3,50 @@ jdk: oraclejdk8
 env:
   global:
   - secure: fMBUkSsCBTw/U907w7Xm/yNzw4lq9yt7zFy0JyEZrA543LI7zQRIclIOtsDTUs0bwY+4KI08MNoYPLJ4TQhSJaH4wQ8b9C5TRsqXNfc0V718TcuYqSXqU6VPRVtX46/fCEWv/HAs0ksUKrHIlpWOwLPQpYDcKxwwlUndaakYKJ8=
+
 before_cache:
 - cd ${TRAVIS_BUILD_DIR}/gradle/caches/
 - find . -name "*.lock" -exec rm -rfv {} \;
 - cd ${TRAVIS_BUILD_DIR}
+
 cache:
   directories:
   - "${TRAVIS_BUILD_DIR}/gradle/caches/"
   - "${TRAVIS_BUILD_DIR}/gradle/wrapper/dists/"
+  
 notifications:
   email: false
+
 android:
   components:
   - tools
   - build-tools-25.0.2
   - platform-tools
   - tools
+  - android-22
+  - sys-img-armeabi-v7a-android-22
+  
 install:
 - echo yes | sdkmanager "extras;m2repository;com;android;support;constraint;constraint-layout;1.0.2"
 - echo yes | sdkmanager "extras;m2repository;com;android;support;constraint;constraint-layout-solver;1.0.2"
-- "./gradlew --version"
+- ./gradlew --version
 - sdkmanager --list || true
+
+# Emulator Management: Create, Start and Wait
+before_script:
+- android list device
+- echo no | android create avd --force -n test -t android-22 --abi armeabi-v7a -d "5.4in FWVGA"
```

Comment:
Hmm, this doesn't solve the problem itself, it just works around it :/ What happens, if the demo sitemap changes slightly and the widgets we use in this test are again outside of the view?

Reply:
Is there a check for "exists, but not visible"?

Reply:
It seems there isn't :/ Then this should be fine for now :D